config.json

```diff
@@ -93,13 +163,86 @@
           "/equipment/{equipmentId}",
           "/v1/order_statuses/{orderId}",
           "/v1/status_names"
+        ]
+      }
+    },
+    {
+      "role": {
+        "slug": "manager"
```

Comment:
The manager has access to all endpoints. As well as admin.

Reply:
Removed unnecessary configurations. Thanks.